Package.swift

```diff
@@ -32,5 +32,16 @@ let package = Package(
             name: "LoggingTests",
             dependencies: ["Logging"]
         ),
+        // Due to a compiler bug in parsing/lexing in 5.0,
```

Comment:
I vote that we drop 5.0 support

Reply:
Yeah, honestly it's about time. We can go careful and _just_ drop 5.0, the others we can keep around still.

Reply:
Done, removed the horrible workaround with modules; 
We'll have to drop 5.0 though.